packit_service/worker/jobs.py

```diff
@@ -97,6 +102,47 @@ def process_jobs(self, event: Optional[Any]) -> Dict[str, HandlerResults]:
                     handler.clean()
         return handlers_results
 
+    def process_comment_jobs(self, event: Optional[Any]):
+        handlers_results = {}
+        # packit_command can be `/packit build` or `/packit build <with_args>`
+        (packit_mark, *packit_command) = event.comment.split(maxsplit=3)
+
+        if REQUESTED_PULL_REQUEST_COMMENT != packit_mark:
+            logger.debug(
+                f"This PR comment '{packit_mark}' is not handled by packit-service"
+            )
+            return HandlerResults(success=False, details={})
+
+        # packit has command `copr-build`. But PullRequestCommentAction has enum `copr_build`.
+        packit_action = PullRequestCommentAction[packit_command[0].replace("-", "_")]
+        handler_kls: Any = PULL_REQUEST_COMMENT_HANDLER_MAPPING.get(packit_action, None)
+        if not handler_kls:
+            return HandlerResults(
+                success=False,
+                details={
+                    "msg": (
+                        f"This PR trigger command '{REQUESTED_PULL_REQUEST_COMMENT} "
```

Comment:
this message would be slightly misleading since this event will be triggered for *all* PR comments, even those which are not commands for bots, but text for humans, I'd probably rephrase this to
```python
f"PR comment '{event.comment[:35]}' is not a packit-service command"
```

first 35 chars should be enough to store, we don't need complete comments